lib/operations/components/idempotency.rb

```diff
@@ -22,7 +22,7 @@ def call(params, context)
     if failure
       new_result = result(
         params: params,
-        context: context.merge(failure.failure || {})
+        context: context.merge(failure.failure)
```

Comment:
it will be hard to figure what is failing when someone will return `Failure()` instead of `Failure({})`, maybe should indeed merge `{}` when Failure will return `Unit` ? Dry docs describe `Unit` as a something similar to `nil`. WDYT?

Reply:
I think I would keep it simple for now and watch

Reply:
let's go with that 👍